sqlstore/src/it/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateWorksetDaoIT.java

```diff
@@ -24,15 +26,15 @@
 
 public class HibernateWorksetDaoIT extends AbstractDAOTest {
 
-  @Autowired
```

Comment:
developers manual still explicitly mentions autowire and SessionFactory

Reply:
The developers manual is horribly out of date. It should be updated or deleted, but I think either is out of scope of this PR. The section that mentions `SessionFactory` is from before MISO even used Hibernate